File: workspace/data-model/src/main/java/de/pfiva/data/model/snips/Slot.java
package de.pfiva.data.model.snips;

import java.io.Serializable;

public class Slot implements Serializable {

	private static final long serialVersionUID = 1L;
	
	private Range range;
	private String rawValue;
	private Value value;
	private String entity;
	private String slotName;
	
	public Range getRange() {
		return range;
	}
	public void setRange(Range range) {
		this.range = range;
	}
	public String getRawValue() {
		return rawValue;
	}
	public void setRawValue(String rawValue) {
		this.rawValue = rawValue;
	}
	public Value getValue() {
		return value;
	}
	public void setValue(Value value) {
		this.value = value;
	}
	public String getEntity() {
		return entity;
	}
	public void setEntity(String entity) {
		this.entity = entity;
	}
	public String getSlotName() {
		return slotName;
	}
	public void setSlotName(String slotName) {
		this.slotName = slotName;
	}
	@Override
	public String toString() {
		return "Slot [range=" + range + ", rawValue=" + rawValue + ", value=" + value + ", entity=" + entity
				+ ", slotName=" + slotName + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/snips/Slot.ts
import { Range } from "./Range";
import { Value } from "./Value";

export class Slot {

    private _range: Range;
    private _rawValue: string;
    private _value: Value;
    private _entity: string;
    private _slotName: string;

	public get range(): Range {
		return this._range;
	}

	public set range(value: Range) {
		this._range = value;
	}

	public get rawValue(): string {
		return this._rawValue;
	}

	public set rawValue(value: string) {
		this._rawValue = value;
    }
    
	public get value(): Value {
		return this._value;
	}

	public set value(value: Value) {
		this._value = value;
	}

	public get entity(): string {
		return this._entity;
	}

	public set entity(value: string) {
		this._entity = value;
    }
    
	public get slotName(): string {
		return this._slotName;
	}

	public set slotName(value: string) {
		this._slotName = value;
	}
}
